app/assets/stylesheets/admin.scss

```diff
@@ -118,10 +118,34 @@ label {
     padding-top: var(--su-4);
     padding-bottom: var(--su-4);
   }
+
+  &--align {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+  }
 }
 
 .config-authentication-form {
   display: flex;
   flex-direction: column;
   gap: var(--su-4);
 }
+
+.admin-config-checkmark {
+  opacity: 100;
+  width: var(--su-5);
+  height: var(--su-5);
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--accent-success);
+  border-radius: 100%;
+  border: 2px solid var(--base-inverted);
+  padding: 2px;
```

Comment:
Is it possible to use [a crayons variable](https://storybook.forem.com/?path=/story/5-css-utility-classes-padding--p-1) here instead of `2px`? I know we are trying to use variables rather than hardcoding values/magic numbers unless we have a good reason not to.

Maybe @lisasy knows which variable (`var(--su-1)`) would work best? (Although, personally, I think the icon is easier to read without _any_ padding 😛)

Reply:
@vaidehijoshi I think you'll like it now 😄